src/Common/Select.php

```diff
@@ -612,12 +612,14 @@ public function groupBy(array $spec)
      *
      * @param string $cond The HAVING condition.
      *
+     * @param array ...$bind arguments to bind to placeholders
```

Comment:
... me either. might be a useful issue to raise with them? i agree it is confusing. Is a better alternative `mixed`? Or maybe its better to clarify the documentation a  little.